File: WYNKWeb/src/app/Models/ViewModels/InvestigationPrescriptionViewModel.ts
import { InvPrescription } from "../InvestigationPrescription";
import { InvPrescriptionTran } from "../InvestigationPrescriptionTran";



export class InvestigationPrescription {
  
  InvPrescription: InvPrescription = new InvPrescription();
  InvPsT: Array<InvPrescriptionTran> = [];
  Companyname: string;
}

File: WYNK.Data/Model/ViewModel/InvestigationPrescriptionViewModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace WYNK.Data.Model.ViewModel
{
    public class InvestigationPrescriptionk
    {
        public InvestigationPrescription InvPrescription { get; set; }
        public ICollection<InvestigationPrescriptionTran> InvPsT { get; set; }
        public ICollection<UploadInvestigationPrescription> UploadInvestigationPrescription { get; set; }
        public ICollection<invdtt> invdtt { get; set; }
        public string Companyname { get; set; }
    }


    public class invdtt

    {
        public int speid { get; set; }
        public string spename { get; set; }
        public int invid { get; set; }
        public string invName { get; set; }
        public string dname { get; set; }

    }


}

